### PR TITLE
tasks: cleanup S3 as part of pruning

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
At tasks container shutdown, also take a chance to run image-prune on
the S3 stores.

### CODE REVIEW

File: tasks/cockpit-tasks [hunk 1]
```diff
@@ -80,3 +80,8 @@ done
 # Prune old images
 update_bots
 ./image-prune
+
+# Clean up S3
+for region in eu-central-1 us-east-1; do
+    ./image-prune --s3 "https://cockpit-images.${region}.linodeobjects.com/" || true
```
Comment (martinpitt) [inline tasks/cockpit-tasks:86]: Is that alternative truth just to make the integration tests happy? TBH I'd rather get a "bot crashed" notification if pruning fails. If it's just for the tests, could that rather be guarded by some `[ -e the_secret ]`?

Comment (allisonkarlitskaya) [reply tasks/cockpit-tasks:86]: I was actually thinking about network issues...

Comment (allisonkarlitskaya) [reply tasks/cockpit-tasks:86]: ...but now that you mention it, I've heard that it's better to beg forgiveness than to ask permission.

**Review by martinpitt — APPROVED** [review]
Ack, let's land that to get the cleanup going, and we can fine-tune this.

**Pull request merged**